docker/debian/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM debian:stable@sha256:82f8da149d6d567c74564cccd6f355fb5ade42a958e4cde10a1100eaeb24d42e
+FROM debian:stable@sha256:26878d0d3aa5e1980d6f8060b4af32fc48b8edeb1fc4d2d074a13a04b17c95f2
```

I'm suggesting to use this tag instead of only stable:

```suggestion
FROM debian:stable-20240612@sha256:26878d0d3aa5e1980d6f8060b4af32fc48b8edeb1fc4d2d074a13a04b17c95f2
```

I have difficulty of knowing what is changed when only the digest changes. The `debian:stable-20240612` and `debian:stable` have the same manifest digest. Since we are updating them from renovate, it shouldn't be a problem.
A drawback, but also an advantage, is that rebuilding manually the code with that dockerfile will lead to the same base image being pulled, not a new one. (It is already doing this with the pinned digest).


But even more useful, would be to use the more precise debian tag that we want, and let renovate update them for us. For exemple, is trixie the latest version? (They don't have the same digest for now) Do we want a `-slim` image instead?